src/Illuminate/Container/CircularReferenceException.php

```diff
@@ -0,0 +1,36 @@
+<?php namespace Illuminate\Container;
+
+use Exception;
+
+class CircularReferenceException extends Exception {
+
+	/**
+	 * @var array
+	 */
+	protected $buildStack;
+
+    /**
+     *  Create a new Circular Reference Exception instance
```

Comment:
No, just make it "circular reference exception", just as you'd read it. It's a description. It's totally pointless to just repeat the class name. People already know what the class name is.